graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/tracing/FederatedTracingInstrumentation.java

```diff
@@ -166,14 +166,19 @@ public InstrumentationState createState(InstrumentationCreateStateParameters par
             if (throwable instanceof GraphQLError) {
                 graphQLErrors.add((GraphQLError) throwable);
             } else {
-                GraphqlErrorBuilder errorBuilder = GraphqlErrorBuilder.newError()
-                        .message(throwable.getMessage());
+                String message = throwable.getMessage();
+                if (message != null) {
```

Comment:
I think we should still add the error to the trace, we should should default the message to "(null)" or something.